Todo.elm

```diff
@@ -110,6 +111,14 @@ update action model =
           in
               { model | tasks <- List.map updateTask model.tasks }
 
+      {- UpdateTask updates the description, but we want to know when it was
+      actually submitted, so we can filter out empty descriptions. -}
+      SubmitTask id ->
+          let
+              newModel = update (EditingTask id False) model
+          in
+              { newModel | tasks <- List.filter (\t -> String.length t.description > 0) newModel.tasks }
```

Comment:
Could you throw a `String.trim` into the mix here? The official TodoMVC guidelines require that just whitespaces are treated as empty, too. :)